accord-core/src/main/java/accord/messages/Propagate.java

```diff
@@ -59,13 +61,13 @@
 import static accord.utils.Invariants.illegalState;
 
 // TODO (required): detect propagate loops where we don't manage to update anything but should
-public class Propagate implements EpochSupplier, LocalRequest<Status.Known>
+public class Propagate implements EpochSupplier, LocalRequest<Status.Known>, PreLoadContext, MapReduceConsume<SafeCommandStore, Void>
```

Comment:
this was the only `LocalRequest` and details of it leaked into the `LocalRequest` API and made it harder to add more.  I had to remove `PreLoadContext` and `MapReduceConsume` to do what I wanted for bootstrap steps, so had to pull all those into this class; hence the changes